lib/bindings/lookup_payee_bindings.dart

```diff
@@ -1,11 +1,17 @@
 import 'package:get/get.dart';
+import 'package:pispapp/config/config.dart';
 import 'package:pispapp/controllers/ephemeral/lookup_payee_controller.dart';
-import 'package:pispapp/repositories/stubs/stub_account_repository.dart';
+import 'package:pispapp/repositories/stubs/stub_transaction_repository.dart';
+import 'package:pispapp/repositories/transaction_repository.dart';
 
 class LookupPayeeBinding implements Bindings {
   @override
   void dependencies() {
-    Get.put<LookupPayeeController>(
-        LookupPayeeController(StubAccountRepository()));
+    if (TRANSACTION_STUB)
```

Comment:
Should we use some kind of dependency injection framework instead?

Reply:
https://github.com/mojaloop/pisp-demo-app-flutter/issues/26